docs/src/prototypes/chatPane/composeMessage.tsx

```diff
@@ -21,6 +21,7 @@ class ComposeMessage extends React.Component {
       <Input
         fluid
         placeholder="Type a message"
+        input={{ styles: { height: '44px' } }}
```

Comment:
same considerations about `px vs rem` units (https://github.com/stardust-ui/react/pull/326/files#r225954685) apply here as well

Reply:
that would be `3.1428rem` are we sure it's better?

Reply:
yes, as it will correctly apply scaling behavior to this component. At the same time, we might consider to round this value to more readable one, like `3.15rem` - or provide the one you've suggested with comment that it corresponds to 44px for default font size (`16px`)